### ISSUE TITLE
Headless requires JavaFX

### ISSUE THREAD

**Issue by animeavi** [Querz/mcaselector#31]
As the title says, running mcaselector from the command line with "--headless" still requires JavaFX.
AdoptOpenJDK doesn't ship with that atm and I don't think a GUI library is needed to run headless, perhaps move that stuff out of main to another class?

**Comment by animeavi** [comment]
Did some quick digging and one of the first issues is the Color vars in Config.java, those can easily be moved to a new Color.java class or whatever.
However Helper.java has a lot more problems, probably not the last of them.

**Comment by Querz** [comment]
Yes, I tried to prevent that from happening but apparently I forgot the `Color` instances in `Config` and `Tile`.
I refactored and reorganised the contents of toe util package to in the end remove it completely.
See #32 if you want to build and test it.

Running the command to create the image cache files (`--mode cache`) still requires JavaFX though, because it heavily uses the JavaFX `PixelWriter`. It will not run when JavaFX is not installed.

**Comment by Querz** [comment]
Sorry, forgot to remove the static reference to the image that is shown when a region is empty in `Tile`.

**Comment by animeavi** [comment]
Now this  😅
```
Exception in thread "main" java.lang.NoClassDefFoundError: javafx/scene/paint/Paint
        at net.querz.mcaselector.Config.getMinZoomLevel(Config.java:223)
        at net.querz.mcaselector.headless.ParamExecutor.parseAndRun(ParamExecutor.java:68)
        at net.querz.mcaselector.Main.main(Main.java:17)
Caused by: java.lang.ClassNotFoundException: javafx.scene.paint.Paint
        at java.net.URLClassLoader.findClass(URLClassLoader.java:382)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
        at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:349)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
        ... 3 more

```

**Comment by Querz** [comment]
Ok, It looks like the JVM is checking for some JavaFX classes even though the code that uses them in headless mode never gets called. I added a very simple docker setup to easily test OpenJDK and replicate those errors.

I hopefully fixed those issues now.

**Comment by Querz** [comment]
Fixes are in [release 1.8.2](https://github.com/Querz/mcaselector/releases/tag/1.8.2)
